.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
I’ve been of the view that “meta” files like this, and IDE-specific files, don’t belong in the version control system. Probably because they are not part of the product being developed.

I don’t feel strongly about it, though.

Reply:
In my opinion, this is really necessary while there's no better alternative. I'd go even further and add a code style definition for IntelliJ. Ideally, IntelliJ would read our checkstyle rules and configure code style settings to produce complying code but, in the meantime, I think these are very useful.

These things not only help a lot to ease people into start collaborating. They also prevent silly merging conflicts.

I can clearly remember having spent more than one hour monkeying around the code style settings to make the checkstyle rules on Briefcase pass. It was a really frustrating experience that would have been prevented had I have the code style definition already in the repo...

Reply:
OK.